include/zephyr/drivers/fuel_gauge.h

```diff
@@ -84,6 +84,10 @@ extern "C" {
 #define FUEL_GAUGE_SBS_ATRATE_TIME_TO_EMPTY FUEL_GAUGE_SBS_ATRATE_TIME_TO_FULL + 1
 /** AtRateOK (boolean) */
 #define FUEL_GAUGE_SBS_ATRATE_OK	    FUEL_GAUGE_SBS_ATRATE_TIME_TO_EMPTY + 1
+/** Remaining Capacity Alarm (mAh or 10mWh) */
+#define FUEL_GAUGE_REMAINING_CAPACITY_ALARM FUEL_GAUGE_SBS_ATRATE_OK + 1
```

Comment:
Please add `SBS` prefix as this is particularly specific to SBS

Reply:
Refactored it in new revision